.charts.yml

```diff
@@ -0,0 +1,9 @@
+charts:
+  - name: cluster-autoscaler
+    version: 9.29.1
+    repository:
+      url: https://kubernetes.github.io/autoscaler
+  - name: cilium
+    version: 1.15.3
```

Comment:
I agree, but we do not have support for 1.29 and 1.30 yet because of the other issue that is relating to the Keystone auth.. :( so I'd rather keep things as is for now.

Reply:
Agreed. what other issue with keystone auth? I'm running with tests with 1.30 at the moment. Anything I should look at?